deck.md

```diff
@@ -324,6 +295,69 @@ class: branded
 
 # Step 2: Solution /3
 
+```js
+// routes/users.js
+export default async function users(fastify) {
+  fastify.get('/users', {}, async () => [
+    { username: 'alice' },
+    { username: 'bob' },
+  ])
+}
+```
+
+---
+
+# Step 2: Trying it out
+
+#### Note that the / route is now not found
+
+```a
+curl http://localhost:3000/
+{"message":"Route GET:/ not found","error":"Not Found","statusCode":404}
```

Comment:
format it so it's readable without the scroll

![Screenshot 2021-02-02 at 16 16 45](https://user-images.githubusercontent.com/272483/106620663-1355cf80-6572-11eb-9ce4-90979e9a708c.png)